cloudify_rest_client/plugins_update.py

```diff
@@ -106,21 +106,29 @@ def list(self, _include=None, sort=None, is_descending=False, **kwargs):
                                 params=params)
         return self._wrap_list(response)
 
-    def update_plugins(self, blueprint_id, force=False):
+    def update_plugins(self, blueprint_id, force=False,
+                       plugin_name=None, minor=False, minor_except=None):
         """
         Updates the plugins in all the deployments that use the given
         blueprint.
 
         :param blueprint_id: blueprint ID to perform the update with.
         :param force: if to forcefully update when other non-active plugins
          updates exists associated with this blueprint [deprecated].
+        :param plugin_name: list of plugin names to update (only those).
+        :param minor: update all selected plugins to the latest minor release.
+        :param minor_except: list of plugin names; all plugins will be updated
```

Comment:
This looks like a parameter to exclude specific minor versions. Perhaps it should instead be called to_latest?